OrderManagement/src/main/java/com/flmfoods/model/Order.java

```diff
@@ -0,0 +1,29 @@
+package com.flmfoods.model;
+
+import java.util.List;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "orders")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Order {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+	private int userId;
+	private int restaurantId;
+	private String status;
+	private int itemId;
+	private int orderPrice;
+	private List<OrderItem> orderItems;
```

Comment:
```suggestion
	private List<OrderItem> orderItems;
```

Add `@OneToMany` mapping here

Reply:
Added the mapping